tools/clang/test/HLSLFileCheck/validation/callgraph/deriv-in-nested-fn-vs-lib68.hlsl

```diff
@@ -0,0 +1,60 @@
+// RUN: %dxc -T lib_6_8 -Vd %s | %D3DReflect %s | %FileCheck %s -check-prefixes=RDAT
+
+// Check ShaderCompatInfo for illegal scenario with verttex entry point calling
+// a function using derivatives (through Sample()).
+// Used to generate deriv-in-nested-fn-vs.ll
```

Comment:
This is probably a better description than what I had above except this hlsl file doesn't check that, the generated .ll file does. So maybe put the last line at the top? 
```suggestion
// Used to generate deriv-in-nested-fn-vs.ll which will
// check ShaderCompatInfo for illegal scenario with verttex entry point calling
// a function using derivatives (through Sample()).

```

Reply:
This file checks that the ShaderCompatInfo contains the expected state that the validator will catch, which is why validation is disabled on this file, otherwise we wouldn't have RDAT to check.  I'll try to clarify the comment.